README.md

```diff
@@ -11,7 +11,7 @@ It includes generic application functionality and tests that can be easily exten
 
 ## More information
 * [Live Demo](http://sap.github.io/openui5-worklist-app/test/mockServer.html)
-* [Documentation](https://openui5.hana.ondemand.com/#/topic/a77f2d29299247f8a3e30226507b1765)
+* [Documentation](https://help.sap.com/docs/SAP_FIORI_tools/17d50220bcd848aa854c9c182d65b699/dcd9f97aa8de4adab8270315550f2b23.html)
```

Comment:
I have a German system. When I follow the link, the target page is shown in German. This might be confusing when coming from this English page. Should we add the 

   ?locale=en-US

parameter?

Reply:
@codeworrior I left the `?locale=en-US` part out on purpose to allow users to read the documentation in their preferred language. But I guess in the context of README which is English anyway, I guess it makes sense to keep `?locale=en-US`.

Reply:
I'm okay with both approaches. I would suggest to ask our documentation expert whether there's some guidance for such cases (mixed language support).

Reply:
I'm not aware of any such guidance, but I'd intuitively opt for keeping the language of the initial documentation. It would be presumptuous on our part to make that decision for the user. On closer investigation they will find the option to switch languages themselves, should they so desire.